components/playlist/browser/playlist_service.cc

```diff
@@ -314,15 +338,10 @@ void PlaylistService::AddMediaFilesFromItems(
 
 void PlaylistService::NotifyPlaylistChanged(mojom::PlaylistEvent playlist_event,
                                             const std::string& playlist_id) {
-  VLOG(2) << __func__ << ": params="
-          << PlaylistChangeParams::GetPlaylistChangeTypeAsString(
-                 playlist_event);
-  for (auto& service_observer : service_observers_) {
-    service_observer->OnEvent(playlist_event, playlist_id);
+  VLOG(2) << __func__ << ": params=" << ToString(playlist_event);
```

Comment:
How about this w/o additional `ToString()`?
```suggestion
  VLOG(2) << __func__ << ": params=" << playlist_event;
```

Reply:
Good call. Removed